_data/other.yml

```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://www.ulule.com/
+      twitter: ulule
+      facebook: ulule
```

Comment:
it's doing that based on globalization techniques available for facebook pages. it's the best option to link against